plugins/reactions.py

```diff
@@ -0,0 +1,47 @@
+import codecs
+import json
+import asyncio
+import os
+import random
+from cloudbot import hook
+
+
+deal_with_it_phrases = [ 'Stop complaining, \x02{}\x02, and',
+               'Jesus fuck \x02{}\x02, just',
+               'Looks like \x02{}\x02 needs to',
+               'Ever think that \x02{}\x02 just needs to']
+
+
+@hook.on_start()
+def load_macros(bot):
+    global reaction_macros
+    with codecs.open(os.path.join(bot.data_dir, "reaction_macros.json"), encoding="utf-8") as macros:
+        reaction_macros = json.load(macros)
+
+        
+@hook.command('dwi', 'dealwithit')
+def deal_with_it(text, message):
+    """<nick>- Tell <nick> in the channel to deal with it. Code located in reactions.py"""
+    person_needs_to_deal = text.strip()
+    message('{} {}'.format(random.choice(deal_with_it_phrases).format(person_needs_to_deal), random.choice(reaction_macros['deal_with_it_macros'])))
```

Comment:
This should really not be a single line. Formatting the selection from `deal_with_it_phrases` should be its own line